src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -96,6 +102,12 @@ export default class SimidMapCreative extends BaseSimidCreative {
     const findNearestButton = document.getElementById('findNearest');
     findNearestButton.innerText = FIND_NEAREST_TEMPLATE_TEXT + buttonLabel;
     findNearest.onclick = () => this.prepareCreative_();
+    let self = this;
+    document.addEventListener('keydown', function(event) {
+      if (event.keyCode == KeyCodes.RIGHT || event.keyCode == KeyCodes.ENTER) {
```

Comment:
So if they press right or enter at any time it will trigger the map?  That seems wrong, it should only be if the button is currently focused and they press enter.  If you're focused on a button and press enter, im pretty sure it fires a click event automatically